### ISSUE TITLE
Codespell report for "Lsyncd" (on fossies.org)

### ISSUE THREAD

**Issue by jschleus** [lsyncd/lsyncd#673]
The FOSS server [fossies.org](https://fossies.org) - supporting now also the [Lsyncd](https://fossies.org/linux/lsyncd) project - offers among others a feature named "[Source code misspelling reports](https://fossies.org/features.html#codespell)". Such reports are normally only generated on request, but as Fossies administrator I have just created such a [codespell](https://github.com/codespell-project/codespell) based analysis for the "Lsyncd" development version (animated by some fixed typos in the new release):

&nbsp;https://fossies.org/linux/test/lsyncd-master.tar.gz/codespell.html

That version-independent URL should redirect always to the report for the latest "master" version identified by the short GitHub commit ID and a year-month-day string (YYMMDD) representing the according git pull date (mostly = commit date). The data are residing within a special restricted "test" folder that isn't really integrated into the standard Fossies services and should also not be accessible to search engines. The report should be available at least for some weeks and is continuously updated (currently with a half-hourly refresh rate).

By the way, the context type of the spelling errors and typos are marked by a single character (within the sortable column "T"): The less interesting, probably not user visible ones (in "comments") are greyed, while the probably more interesting ones are black-colored. Especially interesting may be unassigned errors marked by a "?" since they may be contained in the source code itself. Some more according information is offered by a tooltip if the mouse is over the string "error context type character" above the table, over the "T" in the table header or over such a context character itself within the table. If JavaScript is enabled all table columns are sortable.

Although after a first review some obviously wrong matches ("False Positives" = FPs) are already filtered out (ignored) please inform me if you find more of them so that I can exclude them if applicable.

Just for information there are also three supplemental pages showing some used "codespell" [configuration details](https://fossies.org/linux/test/lsyncd-master.tar.gz/codespell_conf.html), all obvious [false positives](https://fossies.org/linux/test/lsyncd-master.tar.gz/codespell_fps.html) and a [misspelling history](https://fossies.org/linux/test/lsyncd-master.tar.gz/codespell_hist.html) (log).

OK, spelling corrections certainly have a low priority, but they may also contribute to the overall quality of a software project. So I hope that the report can nevertheless be a little bit useful.

**Comment by poelzi** [comment]
sure. a pr is also welcome. easy fixes for your first commit ;)

**Comment by jschleus** [comment]
Yes, the desire is understandable but I'm too stupid to do a PR :-(

Additionally see the according short passage [codespell_related_misspelling_patches_or_PRs_are_not_done_by_Fossies]( https://fossies.org/features.html#codespell_related_misspelling_patches_or_PRs_are_not_done_by_Fossies)
